### PR TITLE
Polish: strings translated

### PR DESCRIPTION
**Pull request by kejn**
- [x] Polish language support
- [x] *no_pull_requests* string for all translations
- [x] replaced three dots "..." with ellipsis in some translations (as suggested by IDE)
- [ ] suggested a few strings to become translatable

Please let me know which of above can be accepted.

### CODE REVIEW

**Comment by dedepete** [comment]
Please, avoid modifying translations if you are not sure if the changes are correct (e.g."Нет Pull Requests" in RU. It's grammatically correct, but does not meet the general translation's style "Pull Requests отсутствуют". Also the "Pull Requests" string has already been translated as "запросы на слияние". So the final string should be "Запросы на слияние отсутствуют").

I agree that some untranslatable strings can be and should be translated without a loss of the meaning, but I don't think that "Gists" should be translated.

**Comment by kejn** [comment]
Thank you for sharing your thoughts. I don't know all the supported languages so I provided strings for *no_pull_requests* to look like before that string was introduced:

`getString("no") + getString("pull_requests")`

Of course I will welcome the feedback from all other contributors who provide the translations for **FastHub**, so that we could benefit with a valid translation for that new string. :) Please post your suggestions here or directly in my commited code. Here are mine, that I am 99% sure of:
- en (default): *No pull requests*
- de: *Keine Pull Requests*
- pl: *Brak pull requestów*

Regarding the "Gists", I would also like to hear what do other translators have to say. Maybe I am not the only one who would like to translate that string. But if I am, I won't argue and make it back untranslatable without any further unnecessary discussion. :)

**Comment by k0shk0sh** [comment]
any updates on this?

**Comment by k0shk0sh** [comment]
Gist shouldn't be translatable, i'm with no_pull_request however this will take time, I encourage you just to ignore it for now, remove that commit and let us merge this.

**Comment by kejn** [comment]
Ok, I will revert "Gists" and create an Issue for *no_pull_requests*, so that once we have all translations we could merge it separately. Agree?

**Comment by dedepete** [comment]
I think that @kejn can add `no_pull_request` at the end of the default `values/strings.xml` now.  
This change will make no problems as the string will be translated separately later.

**Comment by k0shk0sh** [comment]
Do u think we should keep it?

**Comment by dedepete** [comment]
@k0shk0sh I wanted to write yes, but then I found strings `no_open_pull_requests` and `no_closed_pull_requests`. Why won't we make a universal string for all these cases?

**Comment by kejn** [comment]
Could be the same for Issues

**Comment by k0shk0sh** [comment]
True, then please remove gists and keep it that way. Thanks.

Once you make the commit I'll merge it

**Comment by kejn** [comment]
Is it okay now?

File: app/src/main/res/values-cs/strings.xml [hunk 1]
```diff
@@ -264,11 +263,8 @@
     <string name="no_commits">Žádné příspěvky</string>
     <string name="no_contributors">Žádní přispěvatelé</string>
     <string name="no_releases">Žádná vydání</string>
-    <string name="no_closed_issues">Žádné uzavřené problémy</string>
-    <string name="no_opened_issues">Žádné otevřené problémy</string>
     <string name="no_events">Žádné události</string>
-    <string name="no_open_pull_requests">Žádné otevřené požadavky</string>
-    <string name="no_closed_pull_request">Žádné uzavřené požadavky</string>
+    <string name="no_pull_requests">Žádné požadavky</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values-cs/strings.xml]: Pull request shouldn't be translated.

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -284,11 +283,8 @@
     <string name="no_commits">Фиксации отсутствуют</string>
     <string name="no_contributors">Вкладчики отсутствуют</string>
     <string name="no_releases">Сборки отсутствуют</string>
-    <string name="no_closed_issues">Закрытые Тикеты отсутствуют</string>
-    <string name="no_opened_issues">Открытые Тикеты отсутствуют</string>
     <string name="no_events">События отсутствуют</string>
-    <string name="no_open_pull_requests">Запросы на слияние отсутствуют</string>
-    <string name="no_closed_pull_request">Закрытые Запросы на слияние отсутствуют</string>
+    <string name="no_pull_requests">Запросы на слияние отсутствуют</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values-ru/strings.xml]: here too.

Comment (dedepete) [reply app/src/main/res/values-ru/strings.xml]: Pull Requests can be translated into languages without a loss of the meaning. It's a correct variant.

**Comment by kejn** [comment]
Done.

**Comment by dedepete** [comment]
Pull Requests, Issues and other GitHub terms can be translated without a loss of the meaning. It was correct.

**Comment by k0shk0sh** [comment]
@dedepete nope, github terms shouldn't be translated.

**Comment by dedepete** [comment]
@k0shk0sh but i have translated all of them, and so did some other translators.

**Comment by kejn** [comment]
@dedepete I would also appreciate if we could translate at least some of those terms, but as the main goal of this PR was accepted (adding new translation) I can live with that for now.

@k0shk0sh don't you mind, if we move that topic (making some strings 'translatable') to an Issue and ask other developers about their opinion?

**Comment by k0shk0sh** [comment]
@dedepete then this has been slipped away. And that's wrong 🤦‍♂️

_Sent from my ONEPLUS A5010 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by dedepete** [comment]
@k0shk0sh no problem, I'll rework the translation as soon as I finish moving in.

**Comment by dedepete** [comment]
@kejn agreed. We should discuss it with other users.

**Comment by k0shk0sh** [comment]
@dedepete moving to oversea?

_Sent from my ONEPLUS A5010 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by dedepete** [comment]
@k0shk0sh nope, just moving back to my hometown.

**Pull request merged**